cluster/gce/trusty/configure-helper.sh

```diff
@@ -726,7 +726,7 @@ Note: This looks like a development version, which might not be present on GitHu
 If it isn't, the closest tag is at:
   https://github.com/kubernetes/kubernetes/tree/${gitref}
 "
-    gitref="${version//*+/}"
+    gitref=$(echo ${version} | sed -e 's/.*+//')
```

Comment:
For master branch, the GCI code works fine and we have plenty of Jenkins jobs covering it. For trusty code, no, it was broken since one month ago. Even we fix this line in the trusty code in master branch, it does not work. We don't have any auto test covering trusty config in master branch, and we are not trying to making it green.